fontstash.pyx

```diff
@@ -12,6 +12,18 @@ FONS_ALIGN_BASELINE = fs.FONS_ALIGN_BASELINE
 FONS_ZERO_TOPLEFT = fs.FONS_ZERO_TOPLEFT
 FONS_ZERO_BOTTOMLEFT = fs.FONS_ZERO_BOTTOMLEFT
 
+cdef unicode _ustring(basestring s):
+    if type(s) is unicode:
+        return <unicode>s
+    else:
+        return (<bytes>s).decode('utf-8')
+
+cdef bytes _utf8_string(basestring s):
+    if type(s) is unicode:
+        return (<unicode>s).encode('utf-8')
+    else:
+        return <bytes>s
+
```

Comment:
Ok, I see that. But the difference is not clear in `ustring` vs `utf8_string`.  How about `_utf8_enc_str` and `_utf8_dec_str`?